comp/logs/agent/agentimpl/agent_serverless_init.go

```diff
@@ -47,6 +53,16 @@ func (a *logAgent) SetupPipeline(
 	// setup the sole launcher for this agent
 	lnchrs := launchers.NewLaunchers(a.sources, pipelineProvider, a.auditor, a.tracker)
 	lnchrs.AddLauncher(channel.NewLauncher())
+	lnchrs.AddLauncher(filelauncher.NewLauncher(
+		a.config.GetInt("logs_config.open_files_limit"),
+		filelauncher.DefaultSleepDuration,
+		a.config.GetBool("logs_config.validate_pod_container_id"),
+		time.Duration(a.config.GetFloat64("logs_config.file_scan_period")*float64(time.Second)),
+		a.config.GetString("logs_config.file_wildcard_selection_mode"), a.flarecontroller))
+	lnchrs.AddLauncher(listener.NewLauncher(a.config.GetInt("logs_config.frame_size")))
+	lnchrs.AddLauncher(journald.NewLauncher(a.flarecontroller))
+	lnchrs.AddLauncher(windowsevent.NewLauncher())
+	lnchrs.AddLauncher(container.NewLauncher(a.sources, wmeta))
```

Comment:
From the naming, that seems overkill for serverless.
I tried to remove them line by line but they were causing errors somewhere

Reply:
It sounds like a good solution. I tried to build now but the build fails because of some bad merge or conflict
```
../../comp/core/telemetry/telemetryimpl/telemetry.go:66:9: cannot use &telemetryImpl{…} (value of type *telemetryImpl) as telemetry.Component value in return statement: *telemetryImpl does not implement telemetry.Component (wrong type for method Meter)
                have Meter(string, ...telemetry.MeterOption) "go.opentelemetry.io/otel/metric".Meter
                want Meter(string, ...telemetry.MeterOption) telemetry.Meter
../../comp/core/telemetry/telemetryimpl/telemetry.go:103:37: cannot use opts (variable of type []telemetry.MeterOption) as []"go.opentelemetry.io/otel/metric".MeterOption value in argument to t.meterProvider.Meter
../../comp/core/telemetry/telemetryimpl/telemetry.go:252:10: cannot use t.defaultRegistry.Gather() (value of type []*io_prometheus_client.MetricFamily) as []*telemetry.MetricFamily value in return statement
../../comp/core/telemetry/telemetryimpl/telemetry.go:258:9: cannot use registry.Gather() (value of type []*io_prometheus_client.MetricFamily) as []*telemetry.MetricFamily value in return statement
```
I will sync up with @jcstorms1 and decide on next steps, likely rebase everything, fix compilation and try to remove those launchers